parachain/node/src/chain_spec.rs

```diff
@@ -1,13 +1,18 @@
 use sp_core::{Pair, Public, sr25519};
 use artemis_runtime::{
-	AccountId, AuraConfig, BalancesConfig, GenesisConfig, GrandpaConfig,
-	SystemConfig, VerifierConfig, VerifierLightclientConfig, WASM_BINARY, Signature
+	AccountId,
+	AuraConfig, BalancesConfig, GenesisConfig, GrandpaConfig,
+	SystemConfig, VerifierConfig, VerifierLightclientConfig,
+	ETHConfig, ERC20Config,
+	WASM_BINARY, Signature
 };
 use sp_consensus_aura::sr25519::AuthorityId as AuraId;
 use sp_finality_grandpa::AuthorityId as GrandpaId;
 use sp_runtime::traits::{Verify, IdentifyAccount};
 use sc_service::ChainType;
 
+use hex_literal::hex;
```

Comment:
Heads up for a conflict with https://github.com/Snowfork/polkadot-ethereum/pull/166/, depending on who merges first

Reply:
fixed the merge conflicts